hook_test.go

```diff
@@ -83,13 +117,18 @@ var _ = Describe("Hook", func() {
 				err := hook.Subscribe(context.Background(), "http://localhost", "[")
 				Expect(err).To(Succeed())
 				err = hook.DoSend(context.Background(), "message", "test")
-				Expect(err).ToNot(Succeed())
-				Expect(err.Error()).To(Equal("send failed"))
+				Expect(err).To(Succeed())
 			})
 			It("with DoSend", func() {
 				err := hook.DoSend(context.Background(), "message", "*")
+				Expect(err).To(Succeed())
```

Comment:
Same as before

Reply:
It's different here as there is no subscriber 
See https://github.com/w6d-io/hook/blob/ae14f0b4c93864d768011d3536d970c085bcf6e7/hook_test.go#L102